### PR TITLE
Remove unused file(object_manager_integration_test.cc)

### PR DESCRIPTION
**Pull request by ffbin**
## Why are these changes needed?
`object_manager_integration_test.cc` has compile error and we do not use it now. I will remove it. It can be added back if needed later.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by kfstorm — APPROVED** [review]

**Review by WangTaoTheTonic — APPROVED** [review]

**Comment by rkooo567** [comment]
I think this is still used?
Screen Shot 2020-12-19 at 10 00 58 AM

@ericl Can you confirm?

**Review by rkooo567 — CHANGES_REQUESTED** [review]
Temporary change request until the question is resolved.

**Comment by ffbin** [comment]
Hi @ahbone , can you help review this PR, thanks.

**Comment by ahbone** [comment]
Yes, object manager test is still useful.

**Comment by ffbin** [comment]
> Yes, object manager test is still useful.

Is `src/ray/raylet/object_manager_integration_test.cc` still in use? It has compile errors.

**Comment by ahbone** [comment]
> > Yes, object manager test is still useful.
> 
> Is `src/ray/raylet/object_manager_integration_test.cc` still in use? It has compile errors.

Sorry, I'm not quite sure about this one. :(

**Comment by rkooo567** [comment]
@ahbone @ffbin I think we should port this test to run with bazel instead of removing it? Any thought?

**Comment by ffbin** [comment]
> @ahbone @ffbin I think we should port this test to run with bazel instead of removing it? Any thought?

Its test case been tested in `object_manager_test.cc` ?

**Comment by ahbone** [comment]
I checked it and it looks like some of the test cases has been covered. I'm on vocation right now and I'll have another look tonight.

**Comment by ahbone** [comment]
The only thing tested in `object_manager_integration_test.cc` is `Push` which has been covered here https://github.com/ray-project/ray/blob/7ad56826dbab8b718de47fbf2752ba0c81cbd843/src/ray/object_manager/test/object_manager_test.cc#L285

**Comment by ffbin** [comment]
> The only thing tested in `object_manager_integration_test.cc` is `Push` which has been covered here
> 
> https://github.com/ray-project/ray/blob/7ad56826dbab8b718de47fbf2752ba0c81cbd843/src/ray/object_manager/test/object_manager_test.cc#L285

@ahbone @rkooo567 So we can delete `object_manager_integration_test.cc`?

**Comment by ahbone** [comment]
I think so.

File: src/ray/test/run_object_manager_tests.sh [hunk 1]
```diff
@@ -42,6 +42,3 @@ sleep 1s
 bazel run //:redis-cli -- -p 6379 shutdown
 bazel run //:redis-cli -- -p 6380 shutdown
 sleep 1s
```
Comment (raulchen) [inline src/ray/test/run_object_manager_tests.sh:44]: remove this line as well

**Review by raulchen — APPROVED** [review]

**Review by rkooo567 — APPROVED** [review]

**Pull request merged**